test/Makefile.am.inc

```diff
@@ -79,24 +79,20 @@ AM_JS_LOG_FLAGS = \
 	@JASMINE_REPORT_ARGUMENT@ \
 	--no-config \
 	$(NULL)
-LOG_COMPILER = $(top_srcdir)/test/gtester-mock-event-recorder.sh
-AM_LOG_FLAGS = -k --verbose
+LOG_COMPILER = gtester
+AM_LOG_FLAGS = -k --tap --verbose
```

Comment:
Does this work now? I always thought `gtester` didn't pass along the `--tap` argument to the test binary, and you had to have a wrapper script.